cloud-inquisitor/ledger/ledger.go

```diff
@@ -0,0 +1,17 @@
+package ledger
+
+import (
+	cloudinquisitor "github.com/RiotGames/cloud-inquisitor/cloud-inquisitor"
+)
+
+type ActionLedgerEntry interface {
+	Commit(interface{}, cloudinquisitor.PassableResource, string, error) error
+}
+
+func GetActionLedger(platform string) ActionLedgerEntry {
+	if platform == "aws" {
```

Comment:
can we make this a switch statement?

`switch platform{
    case "aws":
         ...
}`